pkg/kev/config/schema.go

```diff
@@ -27,9 +27,10 @@ var ServicesSchema = map[string]interface{}{
 		LabelWorkloadImagePullPolicy:                     map[string]interface{}{"type": "string", "pattern": `^(?i)(IfNotPresent|Always)$`},
 		LabelWorkloadImagePullSecret:                     map[string]interface{}{"type": "string"},
 		LabelWorkloadRestartPolicy:                       map[string]interface{}{"type": "string", "pattern": `^(?i)(Always|OnFailure|Never)$`},
-		LabelWorkloadLivenessProbeType:                   map[string]interface{}{"type": "string", "enum": []string{"none", "command", "http"}},
+		LabelWorkloadLivenessProbeType:                   map[string]interface{}{"type": "string", "enum": []string{"none", "command", "http", "tcp"}},
 		LabelWorkloadLivenessProbeCommand:                map[string]interface{}{"type": "string"},
 		LabelWorkloadLivenessProbeHTTPPort:               map[string]interface{}{"type": "string"},
+		LabelWorkloadLivenessProbeTCPPort:                map[string]interface{}{"type": "string"},
```

Comment:
Not strictly related to this line change but what's the implication to the required fields section below? I guess something you want to address in a follow up PR?

Reply:
yeah, I just created another issue(thought I had before) https://github.com/appvia/kev/issues/409

Reply:
currently only type is mandatory but would be good to fail fast when dependent properties are not provided.